src/PayU/PaymentsApi/AluV3/AluV3.php

```diff
@@ -28,7 +28,7 @@ final class AluV3 implements AuthorizationPaymentsApiClient
      * @var array
      */
     private $aluUrlHostname = [
-        //'ro' => 'https://secure.payu.ro',
+        'ro' => 'https://secure.payu.ro',
         'ru' => 'http://ru.payu.local',
```

Comment:
shouldn't this be the prod url instead of ru.payu.local?

Reply:
this has been fixed in the 'Alu v3 minor fixes' pull request